src/credentials/GroupDataProvider.h

```diff
@@ -306,6 +318,9 @@ class GroupDataProvider
     // Key Sets
     //
 
+    static CHIP_ERROR DeriveOperationalKey(const ByteSpan & epoch_key, const ByteSpan & compressed_fabric_id,
```

Comment:
I can't find any reference of this function. Did you mean `DeriveOperationalCredentials`?

Reply:
Removed unused method.